core/config/project_settings.cpp

```diff
@@ -1579,7 +1579,7 @@ ProjectSettings::ProjectSettings() {
 	// installed by the scripts provided in the repository
 	// (check `misc/scripts/install_d3d12_sdk_windows.py`).
 	// For example, if the script installs 1.613.3, the default value must be 613.
-	GLOBAL_DEF_RST(PropertyInfo(Variant::INT, "rendering/rendering_device/d3d12/agility_sdk_version"), 613);
+	GLOBAL_DEF_RST(PropertyInfo(Variant::INT, "rendering/rendering_device/d3d12/agility_sdk_version", PROPERTY_HINT_RANGE, "0,10000,1,or_greater"), 613);
```

Comment:
I'm guessing the main purpose here was to force a positive number?

A range slider doesn't make much sense for a version number so this should use `no_slider` too most likely. And that makes it a fairly convoluted way or just saying "any non-negative number" but yeah we don't have support for that yet.

Reply:
Added (note that it's `hide_slider`, not `no_slider).